frontend/src/reviews/ReviewsList.tsx

```diff
@@ -6,38 +6,33 @@ import EmptyBlob from 'src/common/EmptyBlob'
 import Loading from 'src/common/Loading'
 import MessageAlert from 'src/MessageAlert'
 import ReviewItem from 'src/reviews/ReviewItem'
-import { Decision, ReviewRequestInterface } from 'types/interfaces'
+import { ReviewRequestInterface } from 'types/interfaces'
 
 type ReviewsListProps = {
-  kind?: 'release' | 'access' | 'all'
+  kind?: 'release' | 'access' | 'all' | 'archived'
 }
 
 export default function ReviewsList({ kind = 'all' }: ReviewsListProps) {
   const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
   const { currentUser, isCurrentUserLoading, isCurrentUserError } = useGetCurrentUser()
   const [filteredReviews, setFilteredReviews] = useState<ReviewRequestInterface[]>([])
 
-  const doesNotContainUserApproval = useCallback(
+  const containsUserApproval = useCallback(
     (review: ReviewRequestInterface) => {
-      return (
-        currentUser &&
-        !review.responses.find(
-          (response) => response.user === `user:${currentUser.dn}` && response.decision === Decision.Approve,
-        )
-      )
+      return currentUser && review.responses.find((response) => response.user === `user:${currentUser.dn}`)
```

Comment:
We're no longer looking for approvals, but instead just looking for a response from the user. Is this a mistake? We could have a "requested changes" response, which this logic now treats as an approval.